openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -69,13 +66,81 @@ def load_subset(
     if not representation:
         return
 
+    # Download representation on site
+    modules_manager = ModulesManager()
+    sync_server = modules_manager.get("sync_server")
+    local_site_id = get_local_site_id()
+    sync_server.add_site(
+        project_name,
+        representation["_id"],
+        local_site_id,
+        priority=99,
+    )
+
+    return representation
+
+
+def wait_for_download(project_name, representations: List[dict]):
+    # Get sync server
+    modules_manager = ModulesManager()
+    sync_server = modules_manager.get("sync_server")
+
+    # Reset timer
+    sync_server.reset_timer()
+
+    # Wait for download
+    local_site_id = get_local_site_id()
+    start = time()  # 5 minutes timeout
+    while (
+        not all(
+            sync_server.is_representation_on_site(
+                project_name, r["_id"], local_site_id
+            )
+            for r in representations
+            if r
+        )
+        and time() - start < 60 * 5
+    ):
+        sleep(5)
+
+
+def load_subset(project_name, representation, loader_type=None):
+    """Load the representation of the last version of subset.
+
+    Args:
+        project_name (str): The project name.
+        loader_type (str, optional): The loader name. Defaults to None.
+
+    Returns:
+        The return of the `load_container()` function.
```

Comment:
I think it would be better  if you indicate what `load_container` returns then. And if possible the type as well.